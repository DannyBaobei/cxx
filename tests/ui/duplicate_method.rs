#[cxx::bridge]
mod ffi {
    extern "Rust" {
        type T;
        fn t_method(&self);
        fn t_method(&self);
    }
}

fn main() {}
