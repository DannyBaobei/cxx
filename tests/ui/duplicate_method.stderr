error: the name `t_method` is defined multiple times
 --> tests/ui/duplicate_method.rs:6:9
  |
6 |         fn t_method(&self);
  |         ^^^^^^^^^^^^^^^^^^^
